au4/src/projectscene/view/clipsview/timeformat.h

```diff
@@ -0,0 +1,25 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+**********************************************************************/
+
+#pragma once
+
+#include "rulerformat.h"
+
+class TimeFormat final : public RulerFormat
+{
+public:
+    static const TimeFormat& Instance();
+    ~TimeFormat() override;
+
+    void SetTickSizes(
+        double units, double& major, double& minor, double& minorMinor, int& mDigits) const override;
+
+    void SetLabelString(
+        QString& s, double d, double units, double minor, int mDigits, TickType tickType) const override;
```

Comment:
I think it’s possible in one line... (in AU3, in my opinion, the code width is too small, I think this is a legacy of many years ago, when there were small monitors with low resolution)